defaults/main.yml

```diff
@@ -34,6 +34,7 @@ nginx_access_log: "{{ nginx_log_path }}/access.log"
 nginx_error_log: "{{ nginx_log_path }}/error.log"
 nginx_conf_path: "/etc/nginx"
 nginx_conf: "{{ nginx_conf_path }}/nginx.conf"
+nginx_conf_template_path: 'templates/nginx'
```

Comment:
As we are using double quotes as a De facto standard, I would prefer to use double quotes here